pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "unasync"]
+build-backend = "setuptools.build_meta"
```

Comment:
probably need to include this file in manifest.in, furthermore you'll need setuptools 40.6.2 or you'll need to also specify setup.py in manifest as to include.

Reply:
Thank you @gaborbernat for going above and beyond and looking at this PR! ✨ This was very helpful, and allowed me to make tox work locally! 💯